index.js

```diff
@@ -57,10 +59,27 @@ ws.on('request', req => {
   });
 });
 
-binance.websockets.candlesticks(['ETHBTC'], "1m", (candlesticks) => {
-  let { e:eventType, E:eventTime, s:symbol, k:ticks } = candlesticks;
-  let { o:open, h:high, l:low, c:close, v:volume, n:trades, i:interval, x:isFinal, q:quoteVolume, V:buyVolume, Q:quoteBuyVolume } = ticks;
-  let prices = JSON.stringify(ticks);
+binance.websockets.candlesticks([CURRENCY], '1m', (candlesticks) => {
+  const { 
+    e:eventType, 
+    E:eventTime, 
```

Comment:
please remove useless vars

Reply:
we will need them later, when we improve our chart. =-\
eventTime - Time when currency was changes, eventType - type of change

Reply:
it's not the best practice, but ok

Reply:
> it's not the best practice, but ok

omg.....this js-developers....fuck...